aptos-move/block-executor/src/txn_last_input_output.rs

```diff
@@ -212,21 +212,28 @@ impl<T: Transaction, O: TransactionOutput<Txn = T>, E: Debug + Send + Clone>
         )
     }
 
-    pub(crate) fn maybe_execution_error(&self, txn_idx: TxnIndex) -> Option<Error<E>> {
+    pub(crate) fn module_rw_intersection(
```

Comment:
Naming (of pub fns): I'd expect this to return the intersection of the reads and writes.

Reply:
changed return types to Result and method name to _ok(), makes more sense this way too.